src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -87,39 +87,41 @@ public static function substr(?string $string, int $from, ?int $length = null):
 
     public static function formatTime(int|float $secs, int $precision = 1): string
     {
+        $ms = $secs * 1000;
         $secs = (int) floor($secs);
 
-        if (0 === $secs) {
-            return '< 1 sec';
+        if ($ms < 0.001) {
```

Comment:
Shouldn't it be `$ms < 1` as `$ms` contains the number of milliseconds, i.e. 1 = 1ms.